rbi/core/hash.rbi

```diff
@@ -689,6 +689,23 @@ class Hash < Object
   # ```
   def fetch_values(*_); end
 
+  # Returns a new Hash excluding entries for the given keys.
+  # [`Hash#except`](https://ruby-doc.org/core-3.0.0/Hash.html#method-i-except)
+  #
+  # ```ruby
+  # h = { a: 100, b: 200, c: 300 }
+  # h.except(:a)          #=> {:b=>200, :c=>300}
+  # ```
+  if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("3.0.0")
```

Comment:
We don't try to differentiate between Ruby versions when writing RBI definitions.  It's true that this leaves us open to accepting code that won't actually run on older Ruby versions, but it hasn't seemed to be a problem in practice.  (Additionally, since RBI files aren't actually executed, this check won't have any effect.)

Please remove the conditional around this definition and re-indent.

Reply:
Thanks for the thorough explanation! Pushed a fix.